server/src/index.js

```diff
@@ -103,18 +103,23 @@ app.get('/api/reports-by-tags/:tag', authorize, async (req, res) => {
   const channelIds = Array.from(new Set(reports.map((r) => r.channel)))
 
 
-  res.json({
+  return {
```

Comment:
When you return the `reports`, let's do some nice [groupBy](https://lodash.com/docs/4.17.15#groupBy) on them before sending them to the UI.